src/components/TopBar/TopBarMenu.vue

```diff
@@ -65,7 +65,7 @@
 			</template>
 
 			<!-- Call layout switcher -->
-			<NcActionButton v-if="showActions && isInCall"
+			<NcActionButton v-if="showCallLayoutSwitch"
```

Comment:
```suggestion
			<NcActionButton v-if="$store.getters.isEmptyCallView"
```
And just move to the template above?